core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -318,6 +326,7 @@ class ReplicaManager(val config: KafkaConfig,
     // A follower can lag behind leader for up to config.replicaLagTimeMaxMs x 1.5 before it is removed from ISR
     scheduler.schedule("isr-expiration", maybeShrinkIsr _, period = config.replicaLagTimeMaxMs / 2, unit = TimeUnit.MILLISECONDS)
     scheduler.schedule("isr-change-propagation", maybePropagateIsrChanges _, period = 2500L, unit = TimeUnit.MILLISECONDS)
+    scheduler.schedule("shutdown-idle-replica-alter-log-dirs-thread", shutdownIdleReplicaAlterLogDirsThread _, period = 2500L, unit = TimeUnit.MILLISECONDS)
```

Comment:
It is named `shutdown-idle-replica-alter-log-dirs-thread` because the thread class is ReplicaAlterLogDirsThread. Do you think we should change the name of the thread class and thread manger class?

Sure, I will update the patch to use 10000L as interval.